src/Components/Sidebar/sidebar.js

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import "../Sidebar/sidebar.css";
+const Sidebar = () => {
+  const [sideBar, setSideBar] = useState(false);
+  return (
+    <>
+      <div className={`sidebar ${sideBar ? "open" : ""}`}>
+        <div className="logo-details">
+          <span>
+            <i className="fa-solid fa-bolt icon"></i>
+          </span>
+          <div className="logo_name">Flasher</div>
+          <i
+            className="bx bx-menu"
+            id="btn"
+            onClick={() => setSideBar((sideBar) => !sideBar)}
+          ></i>
+        </div>
+        <ul className="nav-list">
+          <li>
+            <a href="#">
+              <i class="fa-solid fa-heart"></i>
+              <span className="links_name">Liked</span>
+            </a>
```

Comment:
You'll add routes here in the future right?.. I'm talking about the anchor tags here.

Reply:
yes